extensions/dacpac/src/wizard/pages/deployPlanPage.ts

```diff
@@ -103,6 +106,13 @@ export class DeployPlanPage extends DacFxConfigPage {
 				value: loc.dataLossTextWithCount(result.dataLossAlerts.size)
 			});
 			this.dataLossCheckbox.enabled = true;
+			this.model.potentialDataLoss = true;
+
+			// This will send telemetry event if potential data loss exists
+			TelemetryReporter.createActionEvent(TelemetryViews.DeployPlanPage, 'potentialDataLoss')
```

Comment:
I still think it's going to be hard to use this event correctly. Every time the user enters the page it's going to re-run the populateTable function and possible send off this event.

Do you still need this now that you've added potentialDataLoss to the cancel event?

Reply:
Oops! I thought I removed the telemetry event here, thanks for catching :)